mne/label.py

```diff
@@ -9,16 +9,19 @@
 import os
 import os.path as op
 import copy as cp
+
 import re
 
 import numpy as np
 
+from mne.transforms import apply_trans
```

Comment:
all module imports should be nested (and preferably alphabetical), will push a commit to fix